jupyter_archive/__init__.py

```diff
@@ -14,6 +18,22 @@ def _jupyter_labextension_paths():
     return [{"src": "labextension", "dest": data["name"]}]
 
 
+class JupyterArchive(Configurable):
+    stream_max_buffer_size = Int(help="The max size of tornado IOStream buffer")
```

Comment:
This is required otherwise the variable is not available as configurable.

```suggestion
    stream_max_buffer_size = Int(help="The max size of tornado IOStream buffer", config=True)
```